v20.2/show-schedules.md

```diff
@@ -0,0 +1,129 @@
+---
+title: SHOW SCHEDULES
+summary: The SHOW SCHEDULES statement lists all currently active backup schedules.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> The `SHOW SCHEDULES` [statement](sql-statements.html) lists all of the currently active [backup schedules](create-schedule-for-backup.html).
+
+## Required privileges
+
+By default, only the `root` user can execute `SHOW SCHEDULES`.
+
+## Synopsis
+
+~~~
+SHOW [RUNNING | PAUSED] SCHEDULES [FOR BACKUP];
+SHOW SCHEDULE <schedule_id>;
+~~~
+
+## Parameters
+
+ Parameter      | Description
+----------------+-------------
+`schedule_id`   | The ID of the schedule you want to view.
+
+## Response
+
+The output of `SHOW SCHEDULES` is sorted by creation time. The following fields are returned for each schedule:
+
+Field | Description
+------+------------
+`id` | A unique ID to identify each schedule. This value is used if you want to control schedules (i.e., [pause](pause-schedules.html), [resume](resume-schedules.html), or [drop](drop-schedules.html) it).
+`label` | The name used to identify the backup schedule, given at the time of schedule creation.
+`schedule_status` | The schedule's current status.
+`next_run`  | The [`TIMESTAMP`](timestamp.html) at which the next backup in the schedule is slated to run.
+`state` | Displays last known errors or messages about the backup schedule.
+`recurrence` | How often the backup is taken, which is set at the time of schedule creation.
+`jobsrunning` | The number of [jobs](show-jobs.html) currently running for the schedule.
+`owner` | The [user](create-user.html) who created the backup schedule. Users with active schedules cannot be dropped.
+`created` | The [`TIMESTAMP`](timestamp.html) when the job was created.
+`command` | The [`CREATE SCHEDULE FOR BACKUP`](create-schedule-for-backup.html) statement used
```

Comment:
Nit: It's the command that the schedule will run -- it'd derived from what you entered but it's not actually what you entered on the `CREATE` statement